src/hotspot/share/runtime/osThread.cpp

```diff
@@ -38,18 +38,18 @@ OSThread::~OSThread() {
 
 // Printing
 void OSThread::print_on(outputStream *st) const {
-  st->print("nid=0x%x ", thread_id());
+  st->print("nid=%d ", thread_id());
   switch (_state) {
     case ALLOCATED:               st->print("allocated ");                 break;
     case INITIALIZED:             st->print("initialized ");               break;
     case RUNNABLE:                st->print("runnable ");                  break;
     case MONITOR_WAIT:            st->print("waiting for monitor entry "); break;
     case CONDVAR_WAIT:            st->print("waiting on condition ");      break;
     case OBJECT_WAIT:             st->print("in Object.wait() ");          break;
-    case BREAKPOINTED:            st->print("at breakpoint");               break;
```

Comment:
These cleanups don't seem to have anything to do with this change.

Reply:
Restored.